SpriteKit/MendeluSpriteKit/Game/SceneObjects/Fountain.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Fountain.swift
+//  MendeluSpriteKit
+//
+//  Created by Martina Stremenova on 04.10.2022.
+//
+
+import SpriteKit
+
+final class Fountain: SKSpriteNode {
+    // MARK: Properties
+    private let fountainFlowFrames = SKTextureAtlas(named: Assets.Atlas.fountainFlowing).textures
+
+    private(set) var animations = [String : SKAction]()
+}
+
+// MARK: SceneObject
+extension Fountain: SceneObject {
+    func setup(scene: LevelScene) {
+        setupAction()
+
+        setupFountain()
+    }
+
+    func update(_ currentTime: TimeInterval) {
+        playAnimation(key: Animations.fountainFlowing.rawValue)
+    }
+}
+
+// MARK: AnimatedObject
+extension Fountain: AnimatedObject {}
+
+// MARK: Animations
+private extension Fountain {
+    enum Animations: String {
+        case fountainFlowing
+    }
+}
+
+// MARK: Private API
+private extension Fountain {
+    func setupFountain() {
+        zPosition = Layer.fountain
+    }
+
+    func setupAction() {
+        let timePerFrame: TimeInterval = 0.12
```

Comment:
This property seems unnecessary 🤔

Reply:
Yeah, good catch. I originally thought I'll have more animations. Removing.